src/applications/personalization/profile360/actions/updaters.js

```diff
@@ -46,6 +49,12 @@ export function refreshTransaction(transaction) {
 
       if (isSuccessfulTransaction(transactionRefreshed)) {
         await dispatch(refreshProfile());
+      } else if (isFailedTransaction(transactionRefreshed) && analyticsSectionName) {
+        recordEvent({
+          event: 'profile-edit-failure',
+          'profile-action': 'save-failure',
+          'profile-section': analyticsSectionName,
+        });
       }
```

Comment:
Ah, hate to see conditionals purely for analytics, but not much we can do at this point